app/controllers/spree/admin/taxons_controller.rb

```diff
@@ -113,6 +113,17 @@ def destroy
         @taxon.destroy
         respond_with(@taxon) { |format| format.json { render json: '' } }
       end
+
+      private
+
+      def taxon_params
+        params.require(:taxon).permit(permitted_params)
+      end
+
+      def permitted_params
```

Comment:
Just for consistency... this is different to how you've done it in all the other controllers (two methods instead of one).

Reply:
this is spree code, I'll make it consistent.

Reply:
ok, done, merged into one method only.